### ISSUE TITLE
Thai characters

### ISSUE THREAD

**Issue by azfsone** [ankandrew/fast-plate-ocr#64]
Hi, good work here. I have a question. How did the val results of the model says it achieve 0.99% accuracy on thai plates when the alphabet in the yaml file doesnt even include thai characters?

**Comment by azfsone** [comment]
Also I fine tune a model with new alphabet, the model seems to not be able to read any character outside of the dataset 

fast-plate-ocr train   --model-config-file models/cct_s_v1_global_model_config.yaml --plate-config-file models/cct_s_v1_global_plate_config.yaml   --annotations ./exp2/train_1/labels.csv   --val-annotations ./exp2/val_1/labels.csv   --epochs 30   --batch-size 32   --output-dir trained_models/   --weights-path models/cct_s_v1_global.keras   --label-smoothing 0.0   --weight-decay 0.0005   --lr 0.0005

**Comment by ankandrew** [comment]
Hi, I only used (from all the countries present in val res) plates that contain only latin characters, so for Thailand i.e. these:

![Image](https://github.com/user-attachments/assets/25205c56-e93e-4c81-9cba-af8de3306de8)

And val accuracy is reported only on that subset.

> Also I fine tune a model with new alphabet, the model seems to not be able to read any character outside of the dataset

If you train from scratch, you observe the same? Also not sure if it will help much, but try with `--no-use-ema`.

**Comment by azfsone** [comment]
I tried --no-use-ma doesnt really help,. How many images did you use to train the global model from scratch? I'm guessing due to config changes fine tune might be difficult, so I will try to train from scratch

**Comment by ankandrew** [comment]
Around 4k per country, you probably can you use a bit less and still have good results.
